libs/designsystem/src/lib/scss/base/_typography.scss

```diff
@@ -1,3 +1,9 @@
+@import '~@fontsource/roboto/300.css';
+@import '~@fontsource/roboto/400.css';
+@import '~@fontsource/roboto/500.css';
```

Comment:
I don't see the font-weight 500 in our design system typography?
If added because of styles used in `apps/cookbook/src/app/component-status/component-status.component.scss` consider changing font-weight to a supported weight or alternatively importing the 500 font-weight on the cookbook itself (`apps/cookbook/src/styles.scss`)

Reply:
You're right - I've opted for changing the font-weight in `component-status.component.scss` to 400 in 9da51d3.